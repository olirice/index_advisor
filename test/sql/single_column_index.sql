create extension index_advisor;

select
    optimal_indexes($$ select 1 $$);
