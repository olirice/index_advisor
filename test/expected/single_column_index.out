create extension index_advisor;
ERROR:  extension "index_advisor" already exists
select
    optimal_indexes($$ select 1 $$);
     optimal_indexes      
--------------------------
 create index abc as ...;
(1 row)

